pkg/controller/cassandracluster/reconcile_test.go

```diff
@@ -165,6 +165,83 @@ func TestFlipCassandraClusterUpdateSeedListStatusScaleDC2(t *testing.T) {
 	assert.Equal(true, reflect.DeepEqual(b, status.SeedList))
 }
 
+func TestFlipCassandraClusterUpdateSeedListStatusScaleDC2ManualSeedList(t *testing.T) {
+	assert := assert.New(t)
+
+	_, cc := helperInitCluster(t, "cassandracluster-2DC-ManualSeedlist.yaml")
```

Comment:
I think it makes more sense to build the cluster from an existing sample file as we don't want that option to be used beside multicasskop.

Reply:
Ok it makes sense and it will prevent double update in case of change that affects these files. Will correct asap